gke-kafka-strimzi/terraform/gke-autopilot/main.tf

```diff
@@ -22,7 +22,7 @@ module "network" {
   cluster_prefix = var.cluster_prefix
 }
 
-# [START gke_standard_private_regional_cluster]
+# [START gke_streaming__kafka_strimzi_standard_private_regional_cluster]
```

Comment:
You've got double underscores that should be fixed throughout

Same comment for the other region tags

Reply:
fixed